app/views/filters/study_type/new.html.erb

```diff
@@ -0,0 +1,78 @@
+<%= content_for :page_title, "Filter by study type" %>
+<% content_for :before_content do %>
+  <a href="<%= results_path %>" class="govuk-back-link">Back to search results</a>
+<% end %>
+
+<% if flash[:error] %>
+  <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary" data-ga-event-form="error" data-qa="error">
+    <h2 class="govuk-error-summary__title" id="error-summary-title">
+      There is a problem
+    </h2>
+    <div class="govuk-error-summary__body">
+      <ul class="govuk-list govuk-error-summary__list">
+        <li>
+          <a href="#studytype-error"><%= flash[:error] %></a>
+        </li>
+      </ul>
+    </div>
+  </div>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with url: studytype_path, method: :post do |form| %>
+      <fieldset role="radiogroup" aria-required="true" class="govuk-fieldset">
+        <legend>
+          <h1 class="govuk-heading-xl">Study type</h1>
+        </legend>
+
+        <div class="govuk-form-group <%= flash[:error] ? "govuk-form-group--error" : "" %>">
+          <% if flash[:error] %>
+            <span id="studytype-error" class="govuk-error-message">
+              <span class="govuk-visually-hidden">Error:</span> You must make at least one selection
+            </span>
+          <% end %>
+          <div class="govuk-checkboxes">
+            <div class="govuk-checkboxes__item">
+              <%=
+                form.check_box(
+                  :fulltime,
+                  { class: "govuk-checkboxes__input", data: { qa: 'full_time'}, checked: params[:fulltime] == "True" },
+                  "True",
```

Comment:
Is there a reason you're not using the default boolean values here?

Reply:
We're keeping it in line with the existing Find application - which is `True` and `False` :cry:

Reply:
dont think its case sensitive.... its windows based after all